sechub-solution/docker/SecHub-Debian.dockerfile

```
@@ -13,7 +13,7 @@ ARG BASE_IMAGE
 ARG TARGETARCH
 
 # Build args
-ARG BUILD_TYPE="download"
+ARG BUILD_TYPE="copy"
 
 ARG SECHUB_VERSION="0.35.2"
```

We could force setting the argument by removing the default value:
```suggestion
ARG SECHUB_VERSION
```